### ISSUE TITLE
Keybase: Out of space error

### ISSUE THREAD

**Issue by bboltn** [keybase/client#25072]
I am getting the following error:

```
Keybase: Out of space
Action needed! You are using 168.8GB (157%) of your quota. Please delete some data
```
I am only using 1.27MB of data

I ran `keybase log send` and I ran `keybase fs reset /keybase/team/`

my log id: ee2746c0bba5ab305c485c1c

What can I do to prevent this out of space error from showing in the future?

**Comment by virtadpt** [comment]
I'm seeing the same thing right now.  I'm getting error messages about using 107% of maximum space, but `du -sch keybase/private/drwho/` shows that I'm only using 6.8 gigs.

I've done `keybase log send` as well, but I keep getting "ERROR API network error: Post "https://api-0.core.keybaseapi.com/_/api/1.0/logdump/send.json": net/http: request canceled (Client.Timeout exceeded while awaiting headers)" errors in so doing.

**Comment by bboltn** [comment]
@virtadpt What I ended up doing was completely deleting my keybase group and making a new one.  I might have also reinstalled keybase as well.

**Comment by virtadpt** [comment]
@bboltn Have you seen this happen with a personal (non-group) file store?

**Comment by bboltn** [comment]
No. I’ve only seen it once with a group. ---Brian ***@***.***://keybase.io/bboltnOn Mar 15, 2023, at 10:23 PM, The Doctor ***@***.***> wrote:﻿
@bboltn Have you seen this happen with a personal (non-group) file store?
